### DISCUSSION TITLE
Bulk edit alerts

### DISCUSSION THREAD

**Discussion by gtirloni** [grafana/grafana#65320] [Alerting]
When we migrated from Grafana 7 to 8.x/9.x, the automatic migration of legacy alerts to unified alerting created a `alertname` label that now is causing alerts in our logs: `Mar 24 12:30:09 hostname grafana[770]: logger=ngalert.state.manager t=2023-03-24T12:30:09.986643736-05:00 level=warn msg="Rule declares one or many reserved labels. Those rules labels will be ignored" labels="alertname=alertname"`.

That's fine and we can remove them since that information already exist in the `Rule name`. 

That being said, it's really tiresome to edit 500+ alerts one by one.

It would be awesome if we had some bulk editing capability that allowed us to change certain things in bulk fashion. Example: datasources, labels, appending strings to rule names, etc.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82782. Feel free to continue the discussion around this there. Thank you!
